src/components/app.jsx

```diff
@@ -3,15 +3,16 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import prefixer from 'utils/class-name-prefixer';
 import type { State } from 'state/reducer';
-import Form from './assignment-form';
+// import Form from './assignment-form';
+import Review from './review';
 
 class App extends Component {
 
   render() {
     if (this.props.loggedIn) {
       return (
         <div className={prefixer('container')}>
-          <Form />
+          <Review />
```

Comment:
Should we automatically detect from the crowdsorcerer tag which one we're supposed to render?